src/server/game/Spells/SpellEffects.cpp

```diff
@@ -4678,7 +4678,11 @@ void Spell::EffectLeapBack(SpellEffIndex effIndex)
     float speedxy = m_spellInfo->Effects[effIndex].MiscValue / 10.f;
     float speedz = damage/ 10.f;
     //1891: Disengage
-    unitTarget->JumpTo(speedxy, speedz, m_spellInfo->SpellIconID != 1891);
+   //4022: Rolling Throw
+    m_caster->JumpTo(speedxy, speedz, (m_spellInfo->SpellIconID != 1891 && m_spellInfo->SpellIconID != 4022));
```

Comment:
m_caster should be unitTarget (m_caster becomes also invalid after https://github.com/TrinityCore/TrinityCore/pull/21260)

Reply:
Thank you for the reminder, good to know.